doc/controller_configuration/controller_configuration_tutorial.rst

```diff
@@ -30,146 +79,409 @@ The ``controllers.yaml`` configuration file is located in the ``robot_moveit_con
 
-Or you can make a subscriber with the correct topic name and then ensure that the starting robot state for your move group corresponds to a correct joints angle by using the call back of this subscriber.
+The configuration for these controllers is stored in ``fake_controllers.yaml`` also generated by MSA, for example:
 
-Trajectory Execution Manager Options
-------------------------------------
+.. code-block:: yaml
 
-There are several options for tuning the behavior and safety checks of the execution pipeline in MoveIt. In your ``moveit_config`` package edit the ``trajectory_execution.launch.xml`` file to change the following parameters:
+    controller_list:
+    - name: fake_arm_controller
+        type: $(arg fake_execution_type)
+        joints:
+        - panda_joint1
+        - panda_joint2
+        - panda_joint3
+        - panda_joint4
+        - panda_joint5
+        - panda_joint6
+        - panda_joint7
+    - name: fake_gripper_controller
+        type: $(arg fake_execution_type)
+        joints:
+        - panda_finger_joint1
+        - panda_finger_joint2
+    initial:  # Define initial robot poses per group
+    - group: panda_arm
+        pose: ready
+    - group: panda_hand
+        pose: open
 
- - ``execution_duration_monitoring``: when false, will not throw error is trajectory takes longer than expected to complete at the low-level controller side
- - ``allowed_goal_duration_margin``: Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling)
- - ``allowed_start_tolerance``: Allowed joint-value tolerance for validation that trajectory's first point matches current robot state. If set to zero will skip waiting for robot to stop after execution
+The ``type`` setting specifies the *fake controller interpolation type*:
 
-Example Controller Manager
---------------------------
+* ``interpolate``: performs smooth interpolation between trajectory waypoints - the default for visualization.
+* ``via points``: jumps to the position specified by each trajectory waypoint without interpolation in between - useful for visual debugging.
+* ``last point``: warps directly to the last trajectory waypoint - the fastest method for off-line benchmarking and unit tests.
 
-MoveIt controller managers, somewhat a misnomer, are the interfaces to your custom low level controllers. A better way to think of them are *controller interfaces*. For most use cases, the included :moveit_codedir:`MoveItSimpleControllerManager <moveit_plugins/moveit_simple_controller_manager>` is sufficient if your robot controllers already provide ROS actions for FollowJointTrajectory. If you use *ros_control*, the included :moveit_codedir:`MoveItRosControlInterface <moveit_plugins/moveit_ros_control_interface>` is also ideal.
+Enabling a GripperCommand controller for a ROS controller manager
```

Comment:
This section title is a misnomer. You don't talk about enabling a GripperCommand controller yet. Rather, you enable the gripper to be actuated in Gazebo via the normal `FollowJointTrajectory` API. However, this will not work on the real robot hardware if that just provides a GripperCommand controller.
This is only handled in the next section.

Reply:
I checked my original draft, and I don't have this section, it's something that came in with your changes. I guess I could over-write all the changes and start over with my original draft? Not sure what you want to do here, as this appears to be a comment about your own changes.

Reply:
This section is called "ROS Controllers with Joint Trajectory Action" in my original draft before your changes.
I will rename it back so it doesn't stand out & confuse the flow of the tutorial, then re-read everything again to see if something else needs to be moved around/removed/replaced.